services/addons/images/firmware_manager/sample.env

```diff
@@ -7,8 +7,12 @@ PG_DB_USER=""
 PG_DB_PASS=""
 
 # Blob storage variables
-BLOB_STORAGE_PROVIDER="GCP"
-BLOB_STORAGE_BUCKET=""
+BLOB_STORAGE_PROVIDER=DOCKER
+# GCP Project and Bucket for BLOB storage (if using GCP)
+GCP_PROJECT=
+BLOB_STORAGE_BUCKET=
+# Docker volume mount point for BLOB storage (if using Docker)
+HOST_BLOB_STORAGE_DIRECTORY=./local_blob_storage

```

Comment:
This is fine to use relative path? Maybe let's indicate that more explicitly in the README or here

Reply:
Yes, a relative path can be used. This has been modified to state that explcitly.